.github/scripts/check-template-and-add-labels.ts

```diff
@@ -200,6 +192,25 @@ function extractTemplateTypeFromBody(body: string): TemplateType {
   return TemplateType.None;
 }
 
+// This helper function extracts regression stage (Development, Testing, Production) from bug report issue's body.
+function extractRegressionStageFromBugReportIssueBody(
+  body: string,
+): RegressionStage | undefined {
+  const detectionStageRegex = /### Detection stage\s*\n\s*(.*)/i;
+  const match = body.match(detectionStageRegex);
+  const extractedAnswer = match ? match[1].trim() : undefined;
+
+  if (extractedAnswer === 'On the development branch') {
+    return RegressionStage.Development;
+  } else if (extractedAnswer === 'During release testing') {
+    return RegressionStage.Testing;
+  } else if (extractedAnswer === 'In production') {
+    return RegressionStage.Production;
+  } else {
+    return undefined;
+  }
```

Comment:
Suggestion to make debugg easier in case the detection stage is missing or formatted incorrectly:

```suggestion
  if (!extractedAnswer) {
    throw new Error('Detection stage is missing or formatted incorrectly.');
  }

  if (extractedAnswer === 'On the development branch') {
    return RegressionStage.Development;
  } else if (extractedAnswer === 'During release testing') {
    return RegressionStage.Testing;
  } else if (extractedAnswer === 'In production') {
    return RegressionStage.Production;
  } else {
    throw new Error('Unknown detection stage.');
  }
```

Reply:
Hi @NicolasMassart , have you been able to review my comment?